Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationDataSource.java

```diff
@@ -36,14 +36,17 @@
     private final int dataSourceId;   //< Id in the central repo
     private final String deviceID;  //< Unique to its associated case (not necessarily globally unique)
     private final String name;
+    private final int caseID;
 
 
     CorrelationDataSource(int dataSourceId,
             String deviceID,
-            String name) {
+            String name,
+            int caseID) {
```

Comment:
We should specify if this is the UUID or the database ID. If UUID, then rename it.  Otherwise, just make it clear in the docs.